src/app/app.component.ts

```diff
@@ -307,7 +307,6 @@ export class AppComponent implements OnInit {
    * @param err is command added failed
    */
   onCommandAdded(err) {
-    this.onRefresh();
```

Comment:
@kinfkong  
i am not sure but it should refresh if add command is a success ..isn't it?

Reply:
@standlove

Reply:
Yes, if the data on the UI is affected, the new data should be reloaded.

Reply:
Yes, you are right. However the current refresh doesn't work, check:
https://monosnap.com/file/sdMjNLx4UZSA2ywjiSEa7RTVL9Wkbg
maybe this is related another bug.